sphinx/cmd/quickstart.py

```diff
@@ -454,7 +454,7 @@ def generate(d, overwrite=True, silent=False, templatedir=None):
 
     def write_file(fpath, content, newline=None):
         # type: (unicode, unicode, unicode) -> None
-        if overwrite or not path.isfile(fpath):
+        if overwrite or not path.isfile(fpath) and not d['quiet']:
```

Comment:
This change affects for not only messaging but also writing files.
I think this is broken...

Reply:
Oops. This has got corrupted somehow. I'll fix shortly 😄